fv3core/stencils/c2l_ord.py

```diff
@@ -101,90 +85,15 @@ def compute_ord4(u, v, ua, va, comm, mode=1):
     if mode > 0:
         comm.vector_halo_update(u, v, n_points=utils.halo)
 
-    utmp = utils.make_storage_from_shape(ua.shape, utils.origin)
-    vtmp = utils.make_storage_from_shape(va.shape, utils.origin)
-    j1 = grid.js + 1 if grid.south_edge else grid.js
-    j2 = grid.je - 1 if grid.north_edge else grid.je
-    i1 = grid.is_ + 1 if grid.west_edge else grid.is_
-    i2 = grid.ie - 1 if grid.east_edge else grid.ie
-    vector_tmp(
+    ord4_transform(
```

Comment:
This function isn't needed any more. Can you inline the stencil call into the driver `compute_cubed_to_latlon`?